prescriptor_submission_template.ipynb

```diff
@@ -0,0 +1,392 @@
+    "print(f\"Loaded {len(prescription_indexes)} prescriptions:\")\n",
+    "prescription_indexes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Look at the first prescription plan\n",
+    "idx = 0\n",
+    "idx_df = presc_df[presc_df['PrescriptionIndex'] == idx]\n",
+    "idx_df = idx_df.drop(columns='PrescriptionIndex') # Predictor doesn't need this\n",
+    "idx_df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Save it to a file\n",
+    "presc_x_file = \"prescriptions/2020-08-01_2020-08-04_0.csv\"\n",
+    "idx_df.to_csv(presc_x_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compute prescription plan number of cases\n",
+    "Run the prescription plan through the predictor to get its predicted number of daily cases"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run the predictor\n",
+    "# Assuming the LSTM example has been trained\n",
+    "# TODO: replace by official predictor\n",
+    "pred_x_file = \"predictions/2020-08-01_2020-08-04_0.csv\"\n",
+    "!python covid_xprize/examples/predictors/lstm/predict.py -s {start_date} -e {end_date} -ip {presc_x_file} -o {pred_x_file}\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load the predictions\n",
+    "preds_x_df = pd.read_csv(pred_x_file)\n",
+    "preds_x_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compute prescription plan stringency"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load IP cost weights\n",
+    "cost_df = pd.read_csv(costs_file)\n",
+    "cost_df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "NPI_COLS = ['C1_School closing',\n",
+    "            'C2_Workplace closing',\n",
+    "            'C3_Cancel public events',\n",
+    "            'C4_Restrictions on gatherings',\n",
+    "            'C5_Close public transport',\n",
+    "            'C6_Stay at home requirements',\n",
+    "            'C7_Restrictions on internal movement',\n",
+    "            'C8_International travel controls',\n",
+    "            'H1_Public information campaigns',\n",
+    "            'H2_Testing policy',\n",
+    "            'H3_Contact tracing',\n",
+    "            'H6_Facial Coverings']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def weight_prescriptions_by_cost(pres_df, cost_df):\n",
```

Comment:
Duplicated some code from `prescriptor_robojudge.ipynb` for the sake of example. Probably need to factorize that code in something like `covid_xprize/score/prescriptor_scoring.py`. Later.